src/options.ts

```diff
@@ -214,16 +214,16 @@ export function getCommonCommandOpts() {
 
   const configOpt = createOption(
     '-c, --config <path>',
-    'configuration path (default: synthetics.config.js)'
+    'path to the configuration file (default: synthetics.config.(js|ts))'
   );
 
   const tags = createOption(
     '--tags <name...>',
-    'run/push tests with a tag that matches the glob'
+    'run/push tests with the tag(s) that matches a pattern'
   );
   const match = createOption(
     '--match <name>',
-    'run/push tests with a name or tags that matches the glob'
+    'run/push tests with a name or tags that matches a pattern'
```

Comment:
As a developer I personally find the usage of `glob` to be more specific and helpful than `pattern`, but doesn't matter too much I guess.

Reply:
++ I mainly changed it glob felt more technical than pattern?